src/stores/api/types/eth1.go

```diff
@@ -24,7 +24,7 @@ type UpdateEthAccountRequest struct {
 }
 
 type SignMessageRequest struct {
-	Message hexutil.Bytes `json:"message" validate:"required" example:"0xfeade..." swaggertype:"string"`
+	Message string `json:"message" validate:"required" example:"0x536f6d652064617461 or 536f6d652064617461 (hexaString)" swaggertype:"string"`
```

Comment:
why did you need to change the input type to string if you still expecting a hex value?

Reply:
Because in the end I had the intention to propose accepting human readable messages, but this is out of scope. Messages such as 'Hello world' for instance. That should be supported too.